app/views/users/two_factor_authentication/show.html.erb

```diff
@@ -1,22 +1,10 @@
+<% method = tfa_feature_flag? ? current_user&.second_factor_method : "app" %>
```

Comment:
Can this logic be moved back into the controller rather than in the view?

Reply:
👍